docs/.vuepress/config.yml

```diff
@@ -1,10 +1,40 @@
-title: MESG
+title: MESG Docs
 description: Build applications connecting any technologies
 ga: UA-117583862-3
 # serviceWorker: true
 markdown:
   toc:
     includeLevel: [2]
+head:
+  - - link
```

Comment:
this seems complicated I'm not even sure I understand what is it, it looks like an array of array with one string and one object. 
Don'e we have a better way to have that ?

Reply:
also linter is failing